### ISSUE TITLE
Email notification when a manuscript is marked published

### ISSUE THREAD

**Issue by alicia-cassidy** [dfo-osdt/osp#572]
Automatic email notification to everyone (people that it was shared with, co-authors, managers, admins) when a manuscript is marked as  published (and also to Comms when there was potential public or media interest identified) - should talk to comms to see if there is a generic inbox or person we should contact (regionally or national)

**Comment by vincentauger** [comment]
No problem - for communication, we can decide if we want a generic mailbox or something more sophisticated based on users/regions and a user role. It seems like it may more sense to go with the latter, but we can start with the simple generic mailboxes of all com offices that would be interested.

@alicia-cassidy To clarify, we mean when a "publication" is marked as published from "accepted"?

**Comment by alicia-cassidy** [comment]
Yes, I meant publication! Sounds good for the generic Comms mailbox. We are talking with comms national  on april 10th, we can bring it up there
